draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -461,11 +461,11 @@ When used with databases please refer to {{database_considerations}}.
 The formal definition of the UUID string representation is provided by the following (ABNF) {{RFC5234}}.
 
 ~~~~ abnf
-   UUID                   = 4hexOctet "-"
-                            2hexOctet "-"
-                            2hexOctet "-"
-                            2hexOctet "-"
-                            6hexOctet
+   UUID                   = 4*hexOctet "-"
+                            2*hexOctet "-"
+                            2*hexOctet "-"
+                            2*hexOctet "-"
+                            6*hexOctet
```

Comment:
This is wrong, it says 4 or more, and we need to say exactly 4 octets.
"*" is the wrong modifier.
"4 hexOctet" would actually be correct.
We should use a tool... like "abnfgen"  https://www.quut.com/abnfgen/

Reply:
Also, this ABNF text is duplicated.